src/cart/crt_context.c

```diff
@@ -805,6 +845,10 @@ crt_context_timeout_check(struct crt_context *crt_ctx)
 			  rpc_priv->crp_pub.cr_ep.ep_rank,
 			  rpc_priv->crp_pub.cr_ep.ep_tag);
 
+		if (crt_gdata.cg_use_sensors)
+			(void)d_tm_increment_counter(&crt_ctx->cc_timedout,
+						     count, NULL);
```

Comment:
i think this should be on line 734 instead. some things that are timed out can be renewed

Reply:
fixed.